openshift/pfioh-openshift-template-without-swift.json

```diff
@@ -53,7 +53,9 @@
                 "--forever",
                 "--httpResponse",
                 "--storeBase=/share",
-                "--createDirsAsNeeded"
+                "--createDirsAsNeeded",
+                "--enableTokenAuth",
+                "--tokenPath=/etc/auth/token"
```

Comment:
This dir should be specific to pfioh.  Maybe /etc/pfioh/auth_token

Also, I am confused whether this is an ini or not and what is the name of the file.

Reply:
In this case it's not an ini, it's just a plain text file with a set of key value pairs. However, I did figure out how to create a secret as a .ini, which would require the following syntax:

{
      "apiVersion": "v1",
      "kind": "Secret",
      "metadata": {
        "name": "auth"
      },
      "data":{
      },
      "stringData":{
        "pfiohConfig.ini":"\n[DEFAULT]\nstuff = hi\nthings= bye\n\n[Auth]\ntokens = password, anotherpassword"
      }
    }

Reply:
However, I think this would be syntactically worse

Reply:
Although, for now it could be simplified to this:

{
      "apiVersion": "v1",
      "kind": "Secret",
      "metadata": {
        "name": "auth"
      },
      "data":{
      },
      "stringData":{
        "pfiohConfig.ini":"[Auth]\ntokens = password, anotherpassword"
      }
    }

but keep in mind that escaped newlines will have to precede each key value pair and header we would add, making it hard to read.

Reply:
I would just rather not have two formats.